core/src/main/java/org/elasticsearch/index/shard/ShardPath.java

```diff
@@ -214,6 +240,30 @@ public static ShardPath selectNewPathForShard(NodeEnvironment env, ShardId shard
         return new ShardPath(indexSettings.hasCustomDataPath(), dataPath, statePath, shardId);
     }
 
+    static boolean pathHasEnoughSpace(NodeEnvironment.NodePath path, BigInteger estimatedShardBytes) throws IOException {
```

Comment:
I don't see where this is used? did I miss something or did you not use it?

Reply:
Whoops yeah, this was a leftover, I'll remove it.